src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  background-color: #000;
+  margin: 0;
+}
+
+.block {
+  border: 1px solid #e75551;
+  padding: 10px;
+  box-sizing: border-box;
+}
+
+.main_block {
+  width: 350px;
+  background-color: #308dc0;
+  margin: 20px 20px 40px;
+}
+
+.middle_block {
+  background-color: #ffa500;
+}
+
+.least_block {
```

Comment:
what does `least` here mean?

Reply:
I meant smth kind of the smallest one. Changed it to inner_block. I guess it’s more appropriate.